src/components/Checkbox/Checkbox.tsx

```diff
@@ -3,21 +3,22 @@ import bem from "../../utils/bem";
 import Label, { LabelOptions } from "../Label/Label";
 
 interface CheckboxProps {
-    /** name of the checkbox */
-    name?: string;
-
+    /** Additional attributes passed to the checkbox */
+    attributes?: {};
```

Comment:
These are unused

Reply:
Removing them from the wrapping div into the checkbox `input` based on our conversation yesterday—is that what you mean by unused?

Reply:
aaaah either I was blind yesterday or github is now showing highlighted changes that it wasn't showing yesterday.  It looked like same problem here: https://github.com/NYPL/nypl-design-system/pull/276#discussion_r454680818 but I don't see the problem anymore

Reply:
You're not crazy on all of them! Some of them I pushed up with [cfb38a9](https://github.com/NYPL/nypl-design-system/pull/276/commits/cfb38a952bf4c69d203e4c1b71d4095c789c615f) but some yeah, Github isn't highlighting changes 😱